config/generic-mks-robin-nano-v1.cfg

```diff
@@ -4,11 +4,20 @@
 # configuration setup", select the 28KiB bootloader, and serial (on
 # USART3 PB11/PB10) communication.
 
+# For MKS Robin Nano V1.3 and MKS Robin Nano-S V1.3 boards, the
+# firmware should be compiled for the STM32F1407. When running 
```

Comment:
Typo: perhaps you mean STM32F407

Reply:
Thanks for the comment. It supposed to be STM32F407. I will re-request.